state/state.go

```diff
@@ -906,7 +906,11 @@ func (st *State) FindEntity(tag names.Tag) (Entity, error) {
 	case names.RelationTag:
 		return st.KeyRelation(id)
 	case names.ActionTag:
-		return st.ActionByTag(tag)
+		env, err := st.Model()
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+		return env.ActionByTag(tag)
```

Comment:
I think `FindEntity` will eventually move onto Model too, but this works for now.